src/utils.ts

```diff
@@ -0,0 +1,7 @@
+export function chunk<T>(array: T[], size: number) {
```

Comment:
should add  a basic tests for this chunk function

Reply:
https://github.com/soundxyz/response-cache/pull/41/commits/757d5b2024f157677db521aba0a73be66899e528